README.md

```diff
@@ -182,6 +165,21 @@ the app. To provide consistency we have a convention for these names:
   - **app-e2e**: to run the app with all the other apps necessary to provide
     full end to end user journeys.
 
+`govuk-docker startup` defaults to the `app` stack, but can be overridden in two ways:
```

Comment:
I'd move this into the usage section if needed. The stack section in here is only to document the stack name conventions.